src/asm/rgbasm.5

```diff
@@ -168,7 +175,8 @@ still evaluates both operands of
 and
 .Sq || .
 .Pp
-! returns 1 if the operand was 0, and 0 otherwise.
+.Ic !
```

Comment:
I don't think the `.Ic` will apply correctly, due to `!` being a [delimiter](https://mandoc.bsd.lv/man/mdoc.7.html#Delimiters). `\&!` should work around that, as the link suggests.